pkg/reconciler/revision/resources/deploy.go

```diff
@@ -122,6 +139,11 @@ func makePodSpec(rev *v1.Revision, cfg *config.Config) (*corev1.PodSpec, error)
 		extraVolumes = append(extraVolumes, varTokenVolume)
 	}
 
+	if cfg.Network.QueueProxyCertSecret != "" {
```

Comment:
I think this means changing the `QueueProxtCertSecret` will cause all Knative Serving user deployments to get updated and rolled out, correct?

Reply:
Yes't that's correct.